src/templates/userupdate.html

```diff
@@ -1,40 +1,130 @@
+    <input class="container" type="text" name="major4" value="{{ major4 }}"><br>
+    <label class="container"><b>Enrolled Minors</b></label><br>
+    <input class="container" type="text" name="minor1" value="{{ minor1 }}"><br>
+    <input class="container" type="text" name="minor2" value="{{ minor2 }}"><br>
+    <label class="container"><b>Date of Birth</b></label><br>
+    <input class="container" type="date" name="dob" value="{{ dob }}" required=""><br>
+    <label class="container"><b>Courses Enrolled (Please seperate multiple courses with a comma)</b></label><br>
+    <input class="container" type="text" name="courses" value="{{ courses }}"><br>
+    <input class="container" type="submit" value="submit" class="button"><br>
+    {% raw xsrf_form_html() %}
+  </form>
+  </div>
+{% end %}
+{% if primary_affiliation == 'Faculty' %}
+  <div class="container">
+  <form action="/userinfo/update" method="post">
+    <label class="container"><b>Username</b></label><br>
+    <input class="container" type="text" name="username" value="{{ username }}" readonly><br>
+    <label class="container"><b>Primary Affiliation</b></label><br>
+    <input class="container" type="text" name="primary_affiliation" value="{{ primary_affiliation }}" readonly><br>
+    <label class="container"><b>Email</b></label><br>
+    <input class="container" type="email" name="email" value="{{ email }}" required><br>
+    <label class="container"><b>Date of Birth</b></label><br />
+    <input class="container" type="date" name="dob" value="{{ dob }}" required=""><br />
+    <label class="container"><b>Gender</b></label><br>
+    <select class="container" name="gender" required>
+    {% for user_gender in user_genders %}
+      <option value="{{ user_gender }}"> {{ user_gender }}</option>
+    {% end %}
+    <option selected disabled hidden value=''></option>
+  </select><br/>
+    <label class="container"><b>Ethnicity</b></label><br>
+    <select class="container" name="ethnicity" required>
+    {% for user_ethnicity in user_ethnicities %}
+      <option value="{{ user_ethnicity }}"> {{ user_ethnicity }}</option>
+    {% end %}
+    <option selected disabled hidden value=''></option>
+  </select><br/>
+    <label class="container"><b>Native Language</b></label><br>
+    <select class="container" name="native_language" required>
+    {% for user_native_language in user_native_languages %}
+      <option value="{{ user_native_language }}"> {{ user_native_language }}</option>
+    {% end %}
+    <option selected disabled hidden value=''></option>
+  </select><br/>
+    <label class="container"><b>Departments</b></label><br>
+    <input class="container" type="text" name="departments" value="{{ departments }}" required><br>
+    <label class="container"><b>Courses Instructed</b></label><br>
+    <input class="container" type="text" name="courses" value="{{ courses }}">
+    <div id="div">
+    <button class="container" onclick="appendRow()">+</button><br>
```

Comment:
As for updating the courses... We need to have a course search that will find a class by typeahead name, and let us dynamically add and choose these fields. This is something I believe can be achieved by with a react component that sends ajax requests back to the server talking to [fuzzywuzzy]([fuzzywuzzy](https://github.com/seatgeek/fuzzywuzzy) through some endpoint. This is definitely a different feature, and one I don't think you need to crunch on just yet.

For now, a hacky solution can be to  writing *new* courses to the database. That would be an optimal solution to top off this pr.